controller/persistence/migration_v16.go

```diff
@@ -0,0 +1,55 @@
+package persistence
+
+import (
+	"fmt"
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/foundation/storage/ast"
+	"github.com/openziti/foundation/storage/boltz"
+)
+
+func (m *Migrations) removeOrphanedOttCaEnrollments(step *boltz.MigrationStep) {
+
+	var enrollmentsToDelete []string
+
```

Comment:
You can use ast.BoolNodeTrue as a filter if you want everything

Reply:
Nice. Will do.